media-source/mediasource-remove.html

```diff
@@ -13,7 +13,9 @@
           {
               var sourceBuffer = mediaSource.addSourceBuffer(MediaSourceUtil.AUDIO_VIDEO_TYPE);
 
-              assert_throws("InvalidAccessError", function()
+              mediaSource.duration = 10;
```

Comment:
I see now: this patch includes a new test specifically for *any* remove when duration is NaN, which should throw TypeError (and tests that intend to verify other TypeErrors on remove() need duration to be finite...)